Lib/fontTools/varLib/__init__.py

```diff
@@ -261,7 +262,8 @@ def _SetCoordinates(font, glyphName, coord):
 
 	glyph.recalcBounds(glyf)
 
-	horizontalAdvanceWidth = round(rightSideX - leftSideX)
+	if horizontalAdvanceWidth != 0:
+		horizontalAdvanceWidth = round(rightSideX - leftSideX)
```

Comment:
or `max(round(rightSideX - leftSideX), 0)`

Reply:
Far more elegant. Thanks @anthrotype. Shall I make this amend and squash?